raft/log.go

```diff
@@ -75,8 +76,9 @@ type Log struct {
 	path   string  // data directory
 	config *Config // cluster configuration
 
-	state State         // current node state
-	ch    chan struct{} // state change channel
+	state      State          // current node state
+	initialize chan struct{}  // initialization channel
```

Comment:
I might be missing something but I don't see a value getting sent over this channel anywhere.